app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks += 1
+    if masks > 0:
+        return f"Friends should buy {masks} masks"
+
+    return f"Friends can go to {cafe.name}"
```

Comment:
Good job on using function annotations to specify the type of the function arguments and the return type. This is a good practice and improves code readability.